SiennaOpenAPIModels.jl/test/test-db.jl

```diff
@@ -7,10 +7,18 @@ import SQLite
 import DBInterface
 import Tables
 
+function parse_sqlite_json(s::String)
+    return JSON.parse(s)
+end
+
+function parse_sqlite_json(s)
+    s
+end
```

Comment:
I dont get this

Reply:
I thought it was necessary, but it was a dynamic typing problem in SQLite.